docs/azure.md

```diff
@@ -0,0 +1,75 @@
+# Azure Configuration
```

Comment:
Everything related to Azure is just for our benefit, so we can keep it quite simple.